### PR TITLE
Upload data and string as file using SFTP

### PR DESCRIPTION
**Pull request by rafiki270**

### CODE REVIEW

**Comment by rafiki270** [comment]
@jakeheis any chance you could look at this?

File: Sources/Shout/SFTP.swift [hunk 1]
```diff
@@ -110,6 +110,32 @@ public class SFTP {
     ///   - permissions: the file permissions to create the new file with; defaults to FilePermissions.default
     /// - Throws: SSHError if local file can't be read or upload fails
     public func upload(localURL: URL, remotePath: String, permissions: FilePermissions = .default) throws {
+        let data = try Data(contentsOf: localURL, options: .alwaysMapped)
+        try upload(data: data, remotePath: remotePath, permissions: permissions)
+    }
+    
+    /// Upload a file from the local device to the remote server
```
Comment (jakeheis) [inline Sources/Shout/SFTP.swift]: I think this should read "Upload a string to a file on the remote server" or something like that

File: Sources/Shout/SFTP.swift [hunk 2]
```diff
@@ -110,6 +110,32 @@ public class SFTP {
...
+    ///   - permissions: the file permissions to create the new file with; defaults to FilePermissions.default
+    /// - Throws: SSHError if string is not valid or upload fails
+    public func upload(string: String, remotePath: String, permissions: FilePermissions = .default) throws {
+        guard let data = string.data(using: .utf8) else {
+            throw SSHError.genericError("Unable to convert string to utf8 data")
+        }
+        try upload(data: data, remotePath: remotePath, permissions: permissions)
+    }
+    
+    /// Upload a file from the local device to the remote server
```
Comment (jakeheis) [inline Sources/Shout/SFTP.swift]: I think this should read "Upload data to a file on the remote server" or something like that

**Comment by jakeheis** [comment]
Sorry for the delayed review! A couple doc comments need to be updated then I think this looks good to go

**Comment by rafiki270** [comment]
@jakeheis done

**Comment by jakeheis** [comment]
Thanks for the contribution!

**Pull request merged**